### PR TITLE
ROC-5993 Add upload review order request event

### PR DESCRIPTION
**Pull request by simad**
**Confirm that:**

- [x] *VERSION* has been bumped (only for definition file changes)
- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-5993

### Change description ###

Add event to upload review order pdf in the case documents

**Does this PR introduce a breaking change?**

```
- [ ] Yes
- [x] No
```

### CODE REVIEW

File: definition/data/sheets/AuthorisationCaseEvent.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "TestSupportUpdate", "UserRole": "caseworker-cmc", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DirectionsQuestionnaireDeadline", "UserRole": "caseworker-cmc", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "SealedClaimUpload", "UserRole": "caseworker-cmc", "CRUD": "CR"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ReviewOrderUpload", "UserRole": "caseworker-cmc", "CRUD": "CR"},
```
Comment (SanchitaMoj) [inline definition/data/sheets/AuthorisationCaseEvent.json]: It should not have 'C' as this is just citizen action and caseworker should not be able to do this

File: definition/data/sheets/CaseEvent.json [hunk 2]
```diff
@@ -39,6 +39,7 @@
   {"CaseTypeID": "MoneyClaimCase", "ID": "PinGenerationOperations", "Name": "BulkPrint Defendant & Staff ", "Description": "Bulk Print, Defendant & Staff notification", "DisplayOrder": 1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "SendingRPA", "Name": "RPA Request", "Description": "RPA Request", "DisplayOrder": 1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public"},
   {"CaseTypeID": "MoneyClaimCase", "ID": "SealedClaimUpload", "Name": "Sealed claim uploaded", "Description": "Sealed claim uploaded", "DisplayOrder": 1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public"},
+  {"CaseTypeID": "MoneyClaimCase", "ID": "ReviewOrderUpload", "Name": "Review order uploaded", "Description": "Review order uploaded", "DisplayOrder": 1, "PreConditionState(s)": "*", "PostConditionState": "*", "SecurityClassification": "Public"},
```
Comment (SanchitaMoj) [inline definition/data/sheets/CaseEvent.json]: Is this a request document thats uploaded? If yes, can you kindly label it more appropriately like Review order request uploaded

File: definition/data/sheets/AuthorisationCaseEvent.json (hunk 1 above)
Comment (simad) [reply definition/data/sheets/AuthorisationCaseEvent.json]: 👍

File: definition/data/sheets/FixedLists.json [hunk 3]
```diff
@@ -121,6 +121,7 @@
   {"ID": "DocumentType", "ListElementCode": "DEFENDANT_RESPONSE_RECEIPT", "ListElement": "Defendant Response Receipt"},
   {"ID": "DocumentType", "ListElementCode": "CCJ_REQUEST", "ListElement": "County Court Judgment Request"},
   {"ID": "DocumentType", "ListElementCode": "SETTLEMENT_AGREEMENT", "ListElement": "Settlement Agreement"},
+  {"ID": "DocumentType", "ListElementCode": "REVIEW_ORDER", "ListElement": "Review Order"},
```
Comment (SanchitaMoj) [inline definition/data/sheets/FixedLists.json]: Can you please label it as Review order request?

File: definition/data/sheets/CaseEvent.json (hunk 2 above)
Comment (simad) [reply definition/data/sheets/CaseEvent.json]: 👍

File: definition/data/sheets/FixedLists.json (hunk 3 above)
Comment (simad) [reply definition/data/sheets/FixedLists.json]: 👍

File: definition/data/sheets/AuthorisationCaseEvent.json [hunk 4]
```diff
@@ -26,6 +26,7 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "TestSupportUpdate", "UserRole": "caseworker-cmc", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "DirectionsQuestionnaireDeadline", "UserRole": "caseworker-cmc", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "SealedClaimUpload", "UserRole": "caseworker-cmc", "CRUD": "CR"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ReviewOrderUpload", "UserRole": "caseworker-cmc", "CRUD": "R"},
```
Comment (SanchitaMoj) [inline definition/data/sheets/AuthorisationCaseEvent.json]: No caseworker-cmc entry please, as they should not see in history as well

**Pull request merged**